packages/web/app/src/server/slack.ts

```diff
@@ -11,81 +15,112 @@ const SlackIntegration_addSlackIntegration = graphql(/* GraphQL */ `
   }
 `);
 
-const CallBackQuery = z.object({
-  code: z.string({
-    required_error: 'Invalid code',
-  }),
-  state: z.string({
-    required_error: 'Invalid state',
-  }),
-});
-
-const ConnectParams = z.object({
-  organizationSlug: z.string({
-    required_error: 'Invalid organizationSlug',
-  }),
-});
-
-const SlackOAuthv2AccessResponse = z.discriminatedUnion('ok', [
-  z.object({
-    ok: z.literal(true),
-    access_token: z.string(),
-  }),
-  z.object({
-    ok: z.literal(false),
-    error: z.string(),
-  }),
-]);
+const sentryTagsComponentLevel = {
+  component: 'slack',
+};
 
 export function connectSlack(server: FastifyInstance) {
+  // ----------------------------------
+  // Callback
+  // ----------------------------------
+
+  const CallBackQuery = z.object({
+    code: z.string({
+      required_error: 'Invalid code',
+    }),
+    state: z.string({
+      required_error: 'Invalid state',
+    }),
+  });
+
   server.get('/api/slack/callback', async (req, res) => {
+    const sentryTagsRouteLevel = {
+      ...sentryTagsComponentLevel,
+      route: '/api/slack/callback',
+    };
+
     if (env.slack === null) {
-      throw new Error('The Slack integration is not enabled.');
+      const error = new SlackIntegrationErrors.DisabledError({});
+      Sentry.captureException(error, { tags: sentryTagsRouteLevel });
```

Comment:
Do we have a request ID or any other identifiers that we could use to identify why/for who this is failing?